src/app/Api.NoSql/Startup.cs

```diff
@@ -178,5 +182,19 @@ private void ConfigureCors(IServiceCollection services)
                 });
             });
         }
+
+        private void ConfigureCache(IServiceCollection services)
+        {
+            var cacheSettings = new CacheSettings();
+            _configuration.GetSection("Cache").Bind(cacheSettings);
+
+            services.AddStackExchangeRedisCache(options => options.Configuration = cacheSettings.ConnectionString);
+
+            services.AddScoped<ICache, Cache>();
+
+            services
+                .AddHealthChecks()
```

Comment:
Does it work correctly, considering, that `AddHealthChecks()` is called second time?